overlay.py

```diff
@@ -158,6 +167,7 @@ def on_log(self, client, userdata, level, buf):
         print("\nlog: ", buf)
 
     def on_disconnect(self, client, userdata, msg):
+        self.publish_camera_status(dumps({"connected": False}))
```

Comment:
I don't think this can ever do anything - We'll never be able to publish anything once we've disconnected from the broker. Thankfully, `will_set` handles disconnection messages for us.

Reply:
Done.
1c4efdd

I think `will_set` handles unexpected disconnects so is there a situation where we disconnect the client "normally"? I agree that the disconnect publish is not very useful.

Reply:
Wow, I had no idea the last will is only sent after an "abnormal" disconnect, you're completely right! I don't believe we actually ever terminate raspicam in a "graceful" manner, so it should be fine to remove, but at the same time, it would be fine to keep to just in case we ever change that functionality. Since you've already removed it I think that's fine for now, good to keep this in mind though for the future. Good pickup!